dipy/denoise/tests/test_noise_estimate.py

```diff
@@ -49,53 +54,68 @@ def test_piesno():
     assert_(np.abs(sigma - 50) / sigma < 0.03)
 
     sigma = _piesno_3D(rician_noise, N=1, alpha=0.01, l=1, eps=1e-10,
-                         return_mask=False,
-                         initial_estimation=initial_estimation)
+                       return_mask=False,
+                       initial_estimation=initial_estimation)
     assert_(np.abs(sigma - 50) / sigma < 0.03)
 
-    sigma = _piesno_3D(np.zeros_like(rician_noise), N=1, alpha=0.01, l=1, eps=1e-10,
-                     return_mask=False,
-                     initial_estimation=initial_estimation)
+    sigma = _piesno_3D(np.zeros_like(rician_noise), N=1, alpha=0.01, l=1,
+                       eps=1e-10, return_mask=False,
+                       initial_estimation=initial_estimation)
 
     assert_(np.all(sigma == 0))
 
-    sigma, mask = _piesno_3D(np.zeros_like(rician_noise), N=1, alpha=0.01, l=1, eps=1e-10,
-                 return_mask=True,
-                 initial_estimation=initial_estimation)
+    sigma, mask = _piesno_3D(np.zeros_like(rician_noise), N=1, alpha=0.01, l=1,
+                             eps=1e-10, return_mask=True,
+                             initial_estimation=initial_estimation)
 
     assert_(np.all(sigma == 0))
     assert_(np.all(mask == 0))
 
     # Check if no noise points found in array it exits
-    sigma = _piesno_3D(1000*np.ones_like(rician_noise), N=1, alpha=0.01, l=1, eps=1e-10,
-             return_mask=False, initial_estimation=10)
+    sigma = _piesno_3D(1000*np.ones_like(rician_noise), N=1, alpha=0.01, l=1,
+                       eps=1e-10, return_mask=False, initial_estimation=10)
     assert_(np.all(sigma == 10))
 
+
 def test_estimate_sigma():
 
     sigma = estimate_sigma(np.ones((7, 7, 7)), disable_background_masking=True)
     assert_equal(sigma, 0.)
 
-    sigma = estimate_sigma(np.ones((7, 7, 7, 3)), disable_background_masking=True)
+    sigma = estimate_sigma(np.ones((7, 7, 7, 3)),
+                           disable_background_masking=True)
     assert_equal(sigma, np.array([0., 0., 0.]))
 
-    sigma = estimate_sigma(5 * np.ones((7, 7, 7)), disable_background_masking=False)
+    sigma = estimate_sigma(5 * np.ones((7, 7, 7)),
+                           disable_background_masking=False)
     assert_equal(sigma, 0.)
 
-    sigma = estimate_sigma(5 * np.ones((7, 7, 7, 3)), disable_background_masking=False)
+    sigma = estimate_sigma(5 * np.ones((7, 7, 7, 3)),
+                           disable_background_masking=False)
     assert_equal(sigma, np.array([0., 0., 0.]))
 
     arr = np.zeros((3, 3, 3))
     arr[0, 0, 0] = 1
     sigma = estimate_sigma(arr, disable_background_masking=False, N=1)
-    assert_array_almost_equal(sigma, 0.10286889997472792 / np.sqrt(0.42920367320510366))
+<<<<<<< HEAD
```

Comment:
These conflict markers are going to cause some trouble... You might want to run the tests on your machine before you push the code up. You can do that by running `nosetests` in your source-code repository (assuming you've set up you paths correctly).